src/pages/MoviesPage.vue

```diff
@@ -1,10 +1,86 @@
 <template>
-  <MainTitle text="This is Movies Page" />
+<section>
+  <MainTitle text="All the Movies" />
+  <div class="searchBar">
+    <BaseInput  
+      type="text"
+      label="search"
+      placeholder="Search for..."
+      v-model="query"/>
+  </div>
+  <div class="movies-container">
+     
+     <MovieCard
+      v-for="movie in movies"
+      :key="movie.id"
+      :movie="movie"
+    />
+
+   
+  </div>
+
+  
+</section>
+  
 </template>
 
 <script>
 import MainTitle from "../components/MainTitle.vue";
+import MovieCard from '../components/MovieCard.vue';
+import BaseInput from '../components/BaseInput.vue';
+import axios from 'axios';
+
 export default {
-  components: { MainTitle },
+  components: { MainTitle, MovieCard, BaseInput },
+
+  data(){
+  return {
+    movies: [],
+    errorMsg: '',
+    query: '',
+  }
+  },
+
+  computed:{
+    findMovies(){
```

Comment:
`findMovies` sounds more like a name for method and since it's computed value I'd suggest to change to it to something that will indicate that this is ready to use value instead of method. `filteredMovies` maybe?